src/main/java/com/microsoftopentechnologies/windowsazurestorage/service/DownloadFromFileService.java

```diff
@@ -56,20 +58,40 @@ public int execute() {
         return getFilesDownloaded();
     }
 
-    private int scanFileItems(Iterable<ListFileItem> fileItems) throws WAStorageException {
+    public String getPrefix(ListFileItem fileItem) throws URISyntaxException, StorageException {
+        URI fileUri = fileItem.getUri();
+        URI shareUri = fileItem.getShare().getUri();
+        String difference = StringUtils.difference(shareUri.toString(), fileUri.toString());
+        if (difference.startsWith("/")) {
+            difference = difference.substring(1);
```

Comment:
what if multiple layer file share  such as foo1/foo2/..../file.txt?

Reply:
This method compare the target file with the file share(root), multiple layer file will return `
foo1/foo2/..../file.txt`. Add a test case to cover this.